### ISSUE TITLE
Can not support chuangmi.plug.v3

### ISSUE THREAD

**Issue by wowton** [aholstenson/miio#132]
Device token is hidden, cannot add accessory

**Comment by ghost** [comment]
I can get token either, did you find solution?

**Comment by wowton** [comment]
@HatHost I did not resolve still，how did you do?

**Comment by ghost** [comment]
@wowton I find token, but still there is problem with HA, I opened issue please comment if it happens to you: https://github.com/syssi/xiaomiplug/issues/11

EDIT: I got token with NOX method, and midified MiHome app from here:
http://www.kapiba.ru/2017/11/mi-home.html
just sign in and go to network info for token

EDIT2: Now checking seems token was changed by itself, investigating...

**Comment by DeakinWilson** [comment]
yes, I get the same issue with HA and the devices becoming unavailable.  I checked the logs and see the following:

`Got exception while fetching the state: {'code': -30001, 'message': 'Resp invalid json.'}
Components/switch/xiaomi_miio.py (ERROR)
`

**Comment by Dainiakas** [comment]
I have two of these devices 
when already connected to the App and wifi miio is not showing the token, however if you reset the device and connect to it's wifi i can get the Model info as chuangmi.plug.v3 and the token with miio. But this does not help, because as soon as I connect to the home wifi eiter true the app or miio configure the token is not working, and miio discover shows it as unknown device.
Let me know if i can help in any way.

**Comment by tadakatsukami** [comment]
Hi can someone please tell me how they got the token from the chuangmi-plug-v3 I have spent over a day on this and cant find anything. 
Please?
I don't have an android device. Only Iphone.

**Comment by AvailCat** [comment]
> the token is not working

Token will change after connect to Mi Home app or connect to WiFi. You can try install old version of app to read token from network info page or from logs.

There is no too much difference with older model, with an additional command "get_power", and it's output:
```
miio protocol call 192.168.3.117 get_power "[]"
 INFO  Attempting to call get_power on 192.168.3.117
 INFO  Device found, making call
 INFO  Got result:
[
  11500
]
```
This mean 115W (a personal computer)
